README.md

```diff
@@ -1,4 +1,4 @@
-# ![nf-core/tomte](docs/images/nf-core-tomte_logo_light.png#gh-light-mode-only) ![nf-core/tomte](docs/images/nf-core-tomte_logo_dark.png#gh-dark-mode-only)
+# ![tomte](docs/images/tomte_logo_light.png#gh-light-mode-only) ![tomte](docs/images/tomte_logo_dark.png#gh-dark-mode-only)
```

Comment:
```suggestion
# ![tomte](docs/images/tomte_logo_dark.png#gh-light-mode-only) ![tomte](docs/images/tomte_logo_light.png#gh-dark-mode-only)
```

Reply:
Thanks for noticing, I have corrected it now  ^^